app/screens/search/search.js

```diff
@@ -92,7 +92,7 @@ class Search extends Component {
     componentDidUpdate() {
         const {searchingStatus, recent} = this.props;
         const recentLenght = recent.length;
-        const shouldScroll = searchingStatus === RequestStatus.SUCCESS || searchingStatus === RequestStatus.STARTED;
+        const shouldScroll = (searchingStatus === RequestStatus.SUCCESS || searchingStatus === RequestStatus.STARTED) && !this.recentRemoved;
```

Comment:
sounds good but I'm going to need to track the previous prop as in ComponentDidUpdate we don't have access to it as in componentDidReceiveProps, thoughts?